code/modules/mob/living/simple_animal/eldritch_demons.dm

```diff
@@ -16,7 +16,7 @@
 	speed = 0
 	a_intent = INTENT_HARM
 	stop_automated_movement = 1
-	AIStatus = AI_ON
+	AIStatus = AI_OFF
```

Comment:
👀

Reply:
intended, the way summons work would cause the monster to start hitting random stuff.